docs/source/caching/normalized-cache.mdx

```diff
@@ -1,53 +1,84 @@
+sidebar_title: Normalized caches
 ---
 
-> This page focuses on the normalized cache. A normalized cache has several advantages like deduplication and watchers. If you're looking for a simpler albeit coarser cache, take a look at the [HTTP cache](http-cache).
+Apollo Android provides two built-in **normalized caches** for storing and reusing the results of GraphQL operations:
 
+* An [in-memory cache](#in-memory-cache) (`MemoryCache`)
+* A [SQLite-backed cache](#sqlite-cache) (`SqlNormalizedCache`)
 
-## Data Normalization
+You can use one ([or both!](#chaining-caches)) of these caches in your app to improve its responsiveness for most operations.
 
-A normalized cache stores objects by ID. Instead of caching the whole response, the normalized cache can deduplicate common parts. For an example, take these two queries:
+> To get started with a coarser caching strategy that's faster to set up, take a look at the [HTTP cache](./http-cache/).
+
+## What is a normalized cache?
+
+In a GraphQL client, a normalized cache breaks each of your GraphQL operation responses into the individual objects it contains. Then, each object is cached as a _separate entry_ based on its **cache ID**. This means that if multiple responses include the _same_ object, that object can be deduplicated into a single cache entry. This reduces the overall size of the cache and helps keep your cached data consistent and fresh.
+
+You can also use a normalized cache as a single source of truth for your UI, enabling it to react to changes in the cache. To learn more about the normalization process, [see this blog post](https://www.apollographql.com/blog/apollo-client/caching/demystifying-cache-normalization/).
+
+### Normalizing responses
+
+Look at this example query:
 
 ```graphql
-query BookWithAuthorName {
-  favoriteBook {
+query GetFavoriteBook {
+  favoriteBook { # Book object
     id
     title
-    author {
+    author {     # Author object
       id
       name
     }
   }
 }
+```
 
-query AuthorById($id: String!) {
-  author(id: $id) {
-      id
-      name
+This query returns a `Book` object, which in turn includes an `Author` object. An example response from the GraphQL server looks like this:
+
+```json
+{
+  "favoriteBook": {
+    "id": "bk123",
+    "title": "Les Guerriers du silence",
+    "author": {
+      "id": "au456",
+      "name": "Pierre Bordage"
     }
   }
 }
 ```
 
-In the above example, requesting the author of your favorite book with the `AuthorById` query will return a result from the cache if you requested your favorite book before. This works because the author is stored only once in the cache and all the fields were retrieved in the initial `BookWithAuthorName` query. If you were to request more fields, like `birthdate` for example, that wouldn't work anymore.
+A normalized cache does _not_ store this response directly. Instead, it breaks it up into the following entries by default:
+
+```yaml:title=Cache
```

Comment:
Nitpick
```suggestion
```json:title=Cache
```

Reply:
Cool, I wasn't sure whether these cache blocks were going to technically be valid _anything_ so I initially went with the most relaxed syntax 😄